Process Process-1:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 267, in __init__
    super(CrawlerProcess, self).__init__(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 145, in __init__
    self.spider_loader = _get_spider_loader(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 347, in _get_spider_loader
    return loader_cls.from_settings(settings.frozencopy())
  File "/usr/local/lib/python3.6/dist-packages/scrapy/spiderloader.py", line 61, in from_settings
    return cls(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/spiderloader.py", line 25, in __init__
    self._load_all_spiders()
  File "/usr/local/lib/python3.6/dist-packages/scrapy/spiderloader.py", line 47, in _load_all_spiders
    for module in walk_modules(name):
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/misc.py", line 73, in walk_modules
    submod = import_module(fullpath)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/skywalker/Documents/python/roadStatusScrapy/roadStatusScrapy/spiders/Amap.py", line 1
    -*- coding: utf-8 -*-
     ^
SyntaxError: invalid syntax
Process Process-1:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 267, in __init__
    super(CrawlerProcess, self).__init__(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 145, in __init__
    self.spider_loader = _get_spider_loader(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 347, in _get_spider_loader
    return loader_cls.from_settings(settings.frozencopy())
  File "/usr/local/lib/python3.6/dist-packages/scrapy/spiderloader.py", line 61, in from_settings
    return cls(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/spiderloader.py", line 25, in __init__
    self._load_all_spiders()
  File "/usr/local/lib/python3.6/dist-packages/scrapy/spiderloader.py", line 47, in _load_all_spiders
    for module in walk_modules(name):
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/misc.py", line 73, in walk_modules
    submod = import_module(fullpath)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/skywalker/Documents/python/roadStatusScrapy/roadStatusScrapy/spiders/Amap.py", line 1
    -*- coding: utf-8 -*-
     ^
SyntaxError: invalid syntax
Process Process-1:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 267, in __init__
    super(CrawlerProcess, self).__init__(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 145, in __init__
    self.spider_loader = _get_spider_loader(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 347, in _get_spider_loader
    return loader_cls.from_settings(settings.frozencopy())
  File "/usr/local/lib/python3.6/dist-packages/scrapy/spiderloader.py", line 61, in from_settings
    return cls(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/spiderloader.py", line 25, in __init__
    self._load_all_spiders()
  File "/usr/local/lib/python3.6/dist-packages/scrapy/spiderloader.py", line 47, in _load_all_spiders
    for module in walk_modules(name):
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/misc.py", line 73, in walk_modules
    submod = import_module(fullpath)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/skywalker/Documents/python/roadStatusScrapy/roadStatusScrapy/spiders/Amap.py", line 26
    domain = 'https://restapi.amap.com/v3/traffic/status/rectangle?key={key}&level=6&extensions=all&rectangle=')
                                                                                                               ^
SyntaxError: invalid syntax
Process Process-1:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-2:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-2:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-2:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-2:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-3:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-3:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-3:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-4:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-4:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-4:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-5:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
Process Process-5:1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/cmdline.py", line 145, in execute
    cmd.crawler_process = CrawlerProcess(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/crawler.py", line 269, in __init__
    configure_logging(self.settings, install_root_handler)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 101, in configure_logging
    install_scrapy_root_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 111, in install_scrapy_root_handler
    _scrapy_root_handler = _get_handler(settings)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/utils/log.py", line 127, in _get_handler
    handler = logging.FileHandler(filename, encoding=encoding)
  File "/usr/lib/python3.6/logging/__init__.py", line 1032, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.6/logging/__init__.py", line 1061, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/root/python/roadStatus/roadStatusScrapy/roadStatusScrapy/log/roadStatusScrapy_2019_9_24.log'
